slave/build.py

```diff
@@ -251,6 +279,8 @@ def getBuilder(config, path):
         return WebkitBuilder(config, path)
     if os.path.exists(os.path.join(path, "v8", "LICENSE.v8")):
         return V8Builder(config, path)
+    if os.path.exists(os.path.join(path, "mach")):
```

Comment:
I see. I think `components/servo/` would do the job. Changed it to check for that.